common/djangoapps/course_modes/models.py

```diff
@@ -375,6 +403,128 @@ def min_course_price_for_currency(cls, course_id, currency):
         modes = cls.modes_for_course(course_id)
         return min(mode.min_price for mode in modes if mode.currency == currency)
 
+    @classmethod
+    def get_certificate_display(cls, mode, verification_status):
+        """
+         on the basis of mode and verification status make certificate display html strings and css class.
```

Comment:
Could you make docstrings like this one consistent with our formatting? That is, putting a short description on the same line as the opening blockquote, and a more detailed explanation underneath, if necessary:
```
"""Select appropriate display strings and CSS classes.

Uses mode and verification status to select appropriate display strings and CSS classes for certificate display.
```